components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -1461,7 +1461,7 @@ export class GitpodServerEEImpl extends GitpodServerImpl<GitpodClient, GitpodSer
         return repositories;
     }
 
-    async triggerPrebuild(projectId: string, branch: string): Promise<void> {
+    async triggerPrebuild(projectId: string, branchName: string): Promise<void> {
```

Comment:
Nit: I understand that `public` is the default, but maybe not every developer that ever looks at this code will know that. Also, I think it's slightly dangerous to be inconsistent in this file (e.g. if you see both `public async` and `async`, you might accidentally think the second is private by default).

Please either respect the file-local convention (i.e. explicit `public` keyword), or refactor the entire file in a separate commit (i.e. remove all the `public` prefixes).